src/NzbDrone.Core/Notifications/Slack/Slack.cs

```diff
@@ -134,6 +134,23 @@ public override void OnHealthIssue(HealthCheck.HealthCheck healthCheck)
             _proxy.SendPayload(payload, Settings);
         }
 
+        public override void OnHealthRestored(HealthCheck.HealthCheck previousCheck)
+        {
+            var attachments = new List<Attachment>
+                              {
+                                  new Attachment
+                                  {
+                                      Title = "Successful " + previousCheck.Source.Name,
```

Comment:
Same thing here, `Successful` is too generic.

Reply:
I've removed the prefix entirely since Slack uses the message in the CreatePayload beneath to provide context on what the notification is for